packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1328,7 +1328,7 @@ Adds an OAuth 2.0 client application. This application is only available to the
 
 ###### Details
 
-* `issuer_mode` <ApiLifecycle access="ea" /> is visible if the Custom URL Domain feature is enabled. If the feature is enabled, you can set a custom domain URL in the settings for an OpenID Connect token in an app, and this property is returned in the appropriate responses. After the feature is enabled, the default value for new apps is `CUSTOM_URL`. For existing apps, the default remains `ORG_URL`. You can change the value using the API or administrator user interface. To enable the Custom URL Domain feature, contact [Support](https://support.okta.com/help/open_case).
+* `issuer_mode` <ApiLifecycle access="ea" /> is visible if the Custom URL Domain feature is enabled. If the feature is enabled, you can set a custom domain URL in the settings for an OpenID Connect token in an app, and this property is returned in the appropriate responses. After the feature is enabled, the default value for new apps is `CUSTOM_URL`. For existing apps, the default remains `ORG_URL`. You can also update this to `DYNAMIC` if the Custom URL Domain feature is enabled. If it is set to `DYNAMIC` the request for the ID token made from Custom Domain will have the Custom URL as the issuer and the request for the ID token made from Okta Org Domain will have the Org URL as the issuer. You can change the value using the API or administrator user interface. To enable the Custom URL Domain feature, contact [Support](https://support.okta.com/help/open_case).
```

Comment:
Isn't the default issuer mode for new client will always be "DYNAMIC"?

Reply:
Only if the dynamic issuer mode is enabled.

Reply:
updated